src/domain/challenge/common/tabs/AboutSection.tsx

```diff
@@ -49,31 +53,29 @@ interface AboutSectionProps {
   communityId: string | undefined;
   references: ReferenceDetailsFragment[] | undefined;
   metricsItems: MetricItem[];
+  lifecycle?: LifecycleContextTabFragment;
 }
 
 const BLOCK_HEIGHT = 260;
```

Comment:
How? The design clearly states 260px, I can derive it to be around that number, but it won't be that number and it will change when the spacing changes.